lib/Tomcat/ModjkTest.pm

```diff
@@ -0,0 +1,111 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Provide functionality for apache2 apache2-mod_jk installation
```

Comment:
Could you please provide a more explanatory either summary, either comment on top of each function what is the purpose of each?

Reply:
Dee could you check also this point before we merge it? :) It doesn't seem to be handled somewhere.

Reply:
mod_jk_setup() function handles the installation of apache2 and apache2-mod_jk packages.

Reply:
`Provide functionality for apache2 apache2-mod_jk installation and configuration to test the functionality of interaction of apache2 and tomcat via apache2-mod_jk accessing the tomcat manager page, and browsing tomcat examples using keyboard.` 
This still doesn't makes sense for someone who is not familiar with the module :)

Reply:
Anna, I have modified the comment, and let me know is it ok ?

Reply:
Yes, it's much clearer now, thanks :)